base/deprecated.jl

```diff
@@ -1360,6 +1360,9 @@ end
 # issue #25928
 @deprecate wait(t::Task) fetch(t)
 
+# PR #26008
+@deprecate slicedim(A::AbstractArray, d::Integer, i) selectdim(A, d, i)
```

Comment:
If it's a straight renaming of one to the other you can just use `@deprecate slicedim selectdim`

Reply:
Yeah, I suppose it's effectively the same — that would define and export a broader signature, but it'd still hit a method error later on.